### PR TITLE
Expand the ManyVersions error message

### PR DESCRIPTION
**Pull request by francesco086**
Contributes to https://github.com/iterative/gto/issues/377

### CODE REVIEW

File: gto/exceptions.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def __init__(self, name) -> None:
 
 
 class ManyVersions(GTOException):
-    _message = "'{versions}' versions of artifact '{name}' found"
+    _message = "Multiple versions ('{versions}') of artifact '{name}' found on the same git commit. Multiple versions are not allowed."
```
Comment (shcheklein) [inline gto/exceptions.py:86]: nit: Multiple versions are not allowed on the same Git commit. Found <versions> of an artifact <name> on <git ref>

wdyt?

**Review by shcheklein — APPROVED** [review]
A minor comment. Thanks @francesco086 . Overall we can go as is. Let me know.

File: gto/exceptions.py (hunk 1 above)
Comment (francesco086) [reply gto/exceptions.py:86]: Sorry, I am not able to understand your suggestion: "Found of an artifact on"?

Would this be ok?
`Multiple versions are not allowed on the same Git commit. Violation detected for artifact '{name}', versions '{versions}'.`

Comment (francesco086) [reply gto/exceptions.py]: Just in case it is good, I pushed the change.

Comment (shcheklein) [reply gto/exceptions.py]: thanks!

File: ~/Desktop/foo
Comment (pmrowla) [inline ~/Desktop/foo:1]: @francesco086 was this addition intentional?

Comment (francesco086) [reply ~/Desktop/foo:1]: Of course. Not. 😅
Sry about that
